Library/Homebrew/utils.rb

```diff
@@ -770,3 +770,23 @@ def number_readable(number)
   (numstr.size - 3).step(1, -3) { |i| numstr.insert(i, ",") }
   numstr
 end
+
+def truncate_text_to_approximate_size(s, max_bytes, opts = {})
+  front_weight = opts[:front_weight] || 0.5
+  glue = opts[:gloe] || "\n[...snip...]\n"
```

Comment:
`:glue` (typo fixed) = thing that's put between other string elements derived from the truncation. It's not necessary for implementation anywhere now, but this really feels like a place where there should be an external control knob on this function, so I put it in.